src/Effect/Now.purs

```diff
@@ -3,11 +3,13 @@ module Effect.Now
   , nowDateTime
   , nowDate
   , nowTime
+  , getTimezoneOffset
   ) where
 
 import Prelude
 
 import Data.DateTime (Date, DateTime, Time, date, time)
+import Data.Time.Component (Minute)
```

Comment:
This should use the `Minute` from `Data.Time.Duration`: `Component` is for minutes-past-the-hour in a `Time`, rather than a length of time.

Reply:
np, I'll make the change :)